mail-filter/spamassassin/spamassassin-3.4.0-r2.ebuild

```diff
@@ -0,0 +1,217 @@
+	#	(just to be sure, nobody knows how it could happen in the first place).
+	myconf="SYSCONFDIR=${EPREFIX}/etc DATADIR=${EPREFIX}/usr/share/spamassassin"
+
+	# If ssl is enabled, spamc can be built with ssl support
+	if use ssl; then
+		myconf+=" ENABLE_SSL=yes"
+	else
+		myconf+=" ENABLE_SSL=no"
+	fi
+
+	# Set the path to the Perl executable explictly.  This will be used to
+	# create the initial sharpbang line in the scripts and might cause
+	# a versioned app name end in there, see
+	# <https://bugs.gentoo.org/show_bug.cgi?id=62276>
+	myconf+=" PERL_BIN=${EPREFIX}/usr/bin/perl"
+
+	# Add Gentoo tag to make it easy for the upstream devs to spot
+	# possible modifications or patches.
+	#version_tag="g${PV:6}${PR}"
+	#version_str="${PV//_/-}-${version_tag}"
+
+	# Create the Gentoo config file before Makefile.PL is called so it
+	# is copied later on.
+	#echo "version_tag ${version_tag}" > rules/11_gentoo.cf
+
+	# Setting the following env var ensures that no questions are asked.
+	perl-module_src_configure
+	# Configure spamc
+	emake CC="$(tc-getCC)" LDFLAGS="${LDFLAGS}" spamc/Makefile
+}
+
+src_compile() {
+	export PERL_MM_USE_DEFAULT=1
+
+	# Now compile all the stuff selected.
+	perl-module_src_compile
+
+	if use qmail; then
+		emake spamc/qmail-spamc
+	fi
+
+}
+
+src_install () {
+	perl-module_src_install
+
+	# Create the stub dir used by sa-update and friends
+	keepdir /var/lib/spamassassin
+
+	# Move spamd to sbin where it belongs.
+	dodir /usr/sbin
+	mv "${ED}"/usr/bin/spamd "${ED}"/usr/sbin/spamd  || die "move spamd failed"
+
+	if use qmail; then
+		dobin spamc/qmail-spamc
+	fi
+
+	ln -s mail/spamassassin "${ED}"/etc/spamassassin || die
+
+	# Disable plugin by default
+	sed -i -e 's/^loadplugin/\#loadplugin/g' "${ED}"/etc/mail/spamassassin/init.pre || die
+
+	# Add the init and config scripts.
+	newinitd "${FILESDIR}"/3.3.1-spamd.init spamd
+	newconfd "${FILESDIR}"/3.0.0-spamd.conf spamd
+
+	systemd_newunit "${FILESDIR}"/${PN}.service-r1 ${PN}.service
+	systemd_install_serviced "${FILESDIR}"/${PN}.service.conf
+
+	if use postgres; then
+		sed -i -e 's:@USEPOSTGRES@::' "${ED}/etc/init.d/spamd" || die
+
+		dodoc sql/*_pg.sql
+	else
+		sed -i -e '/@USEPOSTGRES@/d' "${ED}/etc/init.d/spamd" || die
+	fi
+
+	if use mysql; then
+		sed -i -e 's:@USEMYSQL@::' "${ED}/etc/init.d/spamd" || die
+
+		dodoc sql/*_mysql.sql
+	else
+		sed -i -e '/@USEMYSQL@/d' "${ED}/etc/init.d/spamd" || die
+	fi
+
+	dodoc NOTICE TRADEMARK CREDITS INSTALL.VMS UPGRADE USAGE \
+		sql/README.bayes sql/README.awl procmailrc.example sample-nonspam.txt \
+		sample-spam.txt spamd/PROTOCOL spamd/README.vpopmail \
+		spamd-apache2/README.apache
+
+	# Rename some docu files so they don't clash with others
+	newdoc spamd/README README.spamd
+	newdoc sql/README README.sql
+	newdoc ldap/README README.ldap
+
+	if use qmail; then
+		dodoc spamc/README.qmail
+	fi
+
+	cp "${FILESDIR}"/secrets.cf "${ED}"/etc/mail/spamassassin/secrets.cf.example || die
```

Comment:
can this line be converted to a doins and moved to line 185, along with fperms, to simplify the ebuild?